### ISSUE TITLE
DEVEOLPING_GUIDE.md needs a section for Windows.

### ISSUE THREAD

**Issue by MooseQuest** [vinyldns/vinyldns#59]
**Is your feature request related to a problem? Please describe.**
It is related to the DEVELOPERS_GUIDE.md

**Describe the solution you'd like**
Developer's guide has instructions for MacOS but not Windows. Should include that for those who don't have access to MacOS.
**Describe alternatives you've considered**

**Additional context**
Information for all platforms allows for many more contributors to help and on-board.

**Comment by pauljamescleary** [comment]
Totally agree.  Need to get my hands on a windows laptop. Having a developer docker image would also be useful. 

Thanks for the great idea!

**Comment by pauljamescleary** [comment]
Sounds like we need a "Doc Request Template" @MooseQuest , "feature request template" probably isnt the best one here.  Will create a template for that as well.

**Comment by nimaeskandary** [comment]
Can the Windows instructions be run vinyldns in WSL bash 🙂

**Comment by mwinsl001c** [comment]
I am ready to put up instructions for the Quickstart only.

**Comment by rebstar6** [comment]
will be merged in after #42 reorganizes the readme vs dev guide

**Comment by britneywright** [comment]
@djbooboo you can resume work on this now. The developer guide is up to date.

**Comment by pauljamescleary** [comment]
Agreed, we should be able to do this.  Someone with windows would need to prototype how it is done
